crypto/cmp/cmp_util.c

```diff
@@ -100,8 +100,8 @@ const char *ossl_cmp_log_parse_metadata(const char *buf,
                     *file = OPENSSL_strndup(p_file, p_line - 1 - p_file);
                     /* no real problem if OPENSSL_strndup() returns NULL */
                     *line = (int)line_number;
-                    msg = strchr(p_level, ':') + 1;
-                    if (*msg == ' ')
+                    msg = strchr(p_level, ':');
+                    if (msg != NULL && *(msg+1) == ' ')
                         msg++;
```

Comment:
This seems to change the existing logic. If I'm reading the code correctly then previously on exit from this function `msg` pointed to the character after `:`, or (if that is a space) the character after that. After your change it points to the `:` character, or (if the character after that is a space) it will point to the space character. That doesn't seem quite right.

Reply:
Yeah you're right, I need a msg++ to increase the address